File: PRAK601-2210817120007-NursyahnaPuteri.py
b, k = map(int, input  ().split ())
isimatrik = list (map (int, input ().split ()))
x = [] 
a = 0

for i in range (b) :
    x.append (isimatrik [a : k + a])
    a = k + a
print ()
for b in x:
    for isimatrik in b:
        print (isimatrik , end=" ")
    print ()

File: PRAK601-2210817120007-NursyahnaPuteri.c
#include <stdio.h>
int main ()
{
    int b, k, i, j, isimatrik;
    scanf ("%d %d",&b, &k);
    int x [b] [k];
    for ( i = 0; i < b; i++) {
        for ( j = 0; j < k; j++){
            scanf ("%d", &isimatrik);
            x [i] [j] = isimatrik;
        }
    }
    for ( i = 0; i < b; i++) {
        for (j = 0; j < k; j++){
            printf ("%d\t", x [i] [j]);
        }
        printf ("\n");
    }   
}